.gitlab-ci.yml

```diff
@@ -35,11 +35,9 @@ lint:
 build:
   stage: build
   extends: .build_image
-  image: $HARBOR/gitlab-ci/docker:20
   variables:
     DF: ".docker/app/Dockerfile --no-cache"
   services:
```

Comment:
@cdoyle-temple I think you should remove this whole services block.  It is already in the gitlab-template provided for us:

Reply:
@sensei100 done

Reply:
Sorry, this is removing too much just the services block.  Lines 41-43. And you should still keep the removal of the image on line 38.

Reply:
@sensei100 -- got it